electron/legendary_utils/library.ts

```diff
@@ -99,17 +129,37 @@ class Library {
     return result
   }
 
+  /**
+   * Update all updateable games.
+   * Uses `listUpdateableGames` along with `LegendaryGame.update`
+   *
+   * @returns Array of results of `Game.update`.
+   */
   public async updateAllGames() {
-    return (await this.listUpdateableGames()).map(LegendaryGame.get).map(
+    return Promise.allSettled((await this.listUpdateableGames()).map(LegendaryGame.get).map(
```

Comment:
Nice! Just need a frontend integration for that then?

Reply:
Yep, this isn't used anywhere as of now.